jujubundlelib/changeset.py

```diff
@@ -276,6 +279,10 @@ def _handle_container_placement(changeset, placement, machine_record_id):
     return container_record_id
 
 
+def _lxd_to_lxc(container_type):
+    return re.sub(r'^lxd$', 'lxc', container_type)
```

Comment:
I'd prefer to not even use re here.

    return 'lxc' if container_type=='lxd' else container_type